custom/conf/app.example.ini

```diff
@@ -372,6 +372,9 @@ INTERNAL_TOKEN=
 ;; Set to true to disable webhooks feature.
 ;DISABLE_WEBHOOKS = false
 ;;
+;; Set to false to disable access tokens feature.
```

Comment:
I'd add a comment here that it should be changed only if using some kind of SSO and that it might break existing integrations.

Reply:
> I'd add a comment here that it should be changed only if using some kind of SSO and that it might break existing integrations.

If someone is touching such settings, should test it before in nonprod env. Describing all possible scenarios here (like SSO/noSSO, only basic auth without tokens, etc.) does not make sense IHMO.